src/time.ts

```diff
@@ -22,7 +24,7 @@ if (typeof setImmediate !== 'undefined') {
 }
 
 function getTimestamp(): number {
```

Comment:
So this used to return millis since epoc and now millis since process started - which I think is fine because we only use it for duration, not a report an actual point in time, but I'll double check that.

Reply:
Yes, that's it.
Also, that is not an integer anymore, but a float.

Reply:
> Also, that is not an integer anymore, but a float.

@aurelien-reeves what do you mean? JavaScript doesn't have an integer or float type, only `number`

Reply:
I mean, before it looked like integers, ie 500ms. With the proposed change, it will look like double, ie 500.11125473917ms.

It may puzzle some users

Reply:
I see what you mean. The reports format the durations as “human friendly” so I don’t think it will make a difference.